packages/@aws-cdk/aws-codebuild/lib/source.ts

```diff
@@ -499,6 +499,17 @@ interface ThirdPartyGitSourceProps extends GitSourceProps {
    * @default every push and every Pull Request (create or update) triggers a build
    */
   readonly webhookFilters?: FilterGroup[];
+
+  /**
+   * The URL that the build will report back to the source provider. Can use built-in CodeBuild variables, like $AWS_REGION
```

Comment:
Multi-line, and full sentences, please:

```suggestion
   * The URL that the build will report back to the source provider.
   * Can use built-in CodeBuild variables, like $AWS_REGION.
```